frontend/apps/picasso/defi/polkadot/hooks/useActiveBonds.ts

```diff
@@ -1,10 +1,11 @@
 import { BondOffer } from "@/stores/defi/polkadot/bonds/types";
 import { useEffect } from "react";
-import { usePicassoProvider } from "@/defi/polkadot/hooks/index";
+import { usePicassoProvider, useSelectedAccount } from "@/defi/polkadot/hooks/index";
```

Comment:
@easteregg did we have a ticket/task for migration to `substrate-react`

Reply:
Not at the moment, will create one.